receiver/m365receiver/metadata.yaml

```
@@ -0,0 +1,205 @@
+    type: string
+    enum: [read, sent, received]
+  outlookApps:
+    name_override: app
+    description: The number of users of Outlook apps.
+    type: string
+    enum: [pop3, imap4, smtp, windows, mac, web, mobile, other_mobile]
+  outlookQuotas:
+    name_override: state
+    description: The types of Outlook quota states.
+    type: string
+    enum: [under_limit, warning, send_prohibited, send_receive_prohibited, indeterminate]
+
+metrics:
+  m365.sharepoint.files.count:
+    enabled: true
+    description: The number of total files across all sites in the last 7 days.
+    unit: "{files}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.sharepoint.files.active.count:
+    enabled: true
+    description: The number of active files across all sites in the last 7 days.
+    unit: "{files}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.sharepoint.sites.active.count:
+    enabled: true
+    description: The number of active sites across SharePoint in the last 7 days.
+    unit: "{files}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.sharepoint.pages.viewed.count:
+    enabled: true
+    description: The number of unique pages viewed across all sites in the last 7 days.
+    unit: "{pages}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.sharepoint.pages.unique.count:
+    enabled: true
+    description: The number of unique views of pages across all sites in the last 7 days.
+    unit: "{views}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.sharepoint.site.storage.count:
+    enabled: true
+    description: The amount of storage used by all sites across SharePoint in the last 7 days.
+    unit: By
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.teams.device_usage.count:
+    enabled: true
+    description: The number of unique users by device/platform that have used Teams in the last 7 days.
+    unit: "{users}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: [teamsDevices]
+  m365.teams.calls.count:
+    enabled: true
+    description: The number of MS Teams calls from users in the organization in the last 7 days.
+    unit: "{calls}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.teams.meetings.count:
+    enabled: true
+    description: The number of MS Teams meetings for users in the organization in the last 7 days.
+    unit: "{meetings}"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: []
+  m365.teams.message.team.count:
```

Just to match the `m365.teams.messages` namespace of the metric below:
```suggestion
  m365.teams.messages.team.count:
```